breathecode/mentorship/tests/urls/tests_academy_mentor.py

```diff
@@ -748,8 +756,59 @@ def test__post__creating_a_element(self):
                 'user_id': 1,
                 'academy_id': 1,
                 'price_per_hour': 20.0,
+                'email': email,
             }),
         ])
 
         mentor_profile = self.bc.database.get('mentorship.MentorProfile', 1, dict=False)
         self.bc.check.queryset_with_pks(mentor_profile.services.all(), [1])
+
+    @patch('django.utils.timezone.now', MagicMock(return_value=UTC_NOW))
+    def test__post__creating_a_element_without_email(self):
+        profile_academy = {
+            'first_name': self.bc.fake.name(),
+            'last_name': self.bc.fake.name(),
+        }
```

Comment:
Add a case when the mentor is being changed and the profile academy doesn't have its property set